.changeset/lucky-mice-know.md

```diff
@@ -0,0 +1,7 @@
+---
+"@getflip/swirl-components": minor
+"@getflip/swirl-components-angular": minor
+"@getflip/swirl-components-react": minor
+---
+
+Extend and align swirl tabs with designs
```

Comment:
Nitpick: Not very meaningful for external partners/devs. I like to state what props I've added or changed, for example.

Reply:
Makes total sense, thanks for pointing that out!
Changed it to: **Adjust swirl tab styles and add justify and padding props**